sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/util/IntervalUtils.scala

```diff
@@ -482,13 +409,19 @@ object IntervalUtils {
       }
     }
 
+    def currentWord: UTF8String = {
+      val strings = s.split(UTF8String.blankString(1), -1)
+      val lenLeft = s.substring(i, s.numBytes()).split(UTF8String.blankString(1), -1).length
```

Comment:
still use `UTF8String` here, because the trim here in `val s = input.trim.toLowerCase` does handle '\n' '\t' ..., If we use `toString` here, we need to additional logic to avoid `ArrayIndexOutOfBoundsException `